op-batcher/batcher/tx_data.go

```diff
@@ -0,0 +1,46 @@
+package batcher
+
+import (
+	"fmt"
+)
+
+type txData struct {
+	frame frameData
+}
+
+// ID returns the id for this transaction data. It can be used as a map key.
+func (td *txData) ID() txID {
+	return td.frame.id
+}
+
+// Bytes returns the transaction data.
+func (td *txData) Bytes() []byte {
+	return append([]byte{0}, td.frame.data...)
```

Comment:
should we add the explanation of []byte{0}?

Reply:
good idea, added some more commentary to `txData` and its methods!